scripts/prepare_vroot_13.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+. scripts/prepare_vroot_functions.sh
+
+PACKAGES_MINIMAL="$PACKAGES_MINIMAL bind918 bind-tools"
+#PACKAGES_MINIMAL=`echo $PACKAGES_MINIMAL | sed 's/quagga/frr8/'`
+PACKAGES_COMMON=`echo $PACKAGES_COMMON | sed 's/lsof//'`
+PACKAGES="$PACKAGES_MINIMAL $PACKAGES_COMMON isc-dhcp44-server isc-dhcp44-client isc-dhcp44-relay \
+    sylpheed apache24 apr db18 jansson xorp netsurf midori wireshark gnome-themes-extra sakura vte3 \
```

Comment:
Are all of these packages really needed? Can we separate default IMUNES packages with these additional ones by creating a new installation target (similar to the "mini" option, something like "extended" which installs $PACKAGES_EXTENDED)? This would mean that we can keep the default IMUNES installation reasonably small, but still having an option to install more packages.